frame/elections-phragmen/src/lib.rs

```diff
@@ -259,6 +248,14 @@ pub mod pallet {
 		#[pallet::constant]
 		type TermDuration: Get<Self::BlockNumber>;
 
+		/// The maximum number of candidates in a phragmen election.
```

Comment:
@kianenigma Should the docs for `MaxCandidates` and `MaxVoters` be more detailed than this?

Reply:
Please add a section warning that the election happens onchain, and these numbers determine how big the election will be. Also note that if the maximum candidates are reached, we don't accept anymore. If max voters are reached, we just ignore voters.